Automattic/vip-go-mu-plugins
# Update/rate limiting stats

## Description

Cleans rate limiting stats, by removing per page ones. Also adds unittests for those

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. It's really hard to recreate the conditions for rate limiting without changing the code.
1. You can change the conditions leading to rate limited
1. Then check `lando -s statsd` for the rate_limited counters
